libs/blocks/global-navigation/global-navigation.css

```diff
@@ -172,6 +172,11 @@ header.global-navigation {
   background-color: var(--feds-background-link--hover--light);
 }
 
+.feds-navItem > .feds-navLink--hoverCaret:hover,
+.feds-navItem > .feds-navLink--hoverCaret:focus {
+  color: var(--feds-color-link--light);
+}
+
```

Comment:
I think this is more complex than it needs to be. `.feds-navLink--hoverCaret` represent top menu items which have a caret, I don't see a need for the `.feds-navItem > ` part.

I also think this belongs to `base.css`, maybe somewhere around [here](https://github.com/adobecom/milo/blob/main/libs/blocks/global-navigation/base.css#L55).

Of course, same comments apply for the desktop styles.